### PR TITLE
Fix ternary expression

### PR DESCRIPTION
**Pull request by geekofcola**
"coordDigest" returns an undefined value, and "verify" doesn't call update method

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "bezier-js",
-  "version": "2.6.0",
+  "version": "2.6.1",
```
Comment (Pomax) [inline package.json]: this is managed by npm, which bumps the version _and_ automatically builds a corresponding git tag, so please don't add this manually =)

Comment (geekofcola) [reply package.json]: oh okay, NP :)

Comment (geekofcola) [reply package.json]: done

Comment (geekofcola) [reply package.json]: Thanks a lot for fixing getLUT :)

**Comment by Pomax** [comment]
v2.6.1 published, thanks for catching this one!

**Pull request merged**